Dockerfile

```diff
@@ -5,5 +5,7 @@ ADD ./templates /etc/confd/templates
 
 VOLUME ["/app/gogs/custom/conf"]
 
+ENV GOGS_SERVICE_ENABLE_REVERSE_PROXY_AUTHENTICATION=false GOGS_SERVICE_ENABLE_REVERSE_PROXY_AUTO_REGISTRATION=false GOGS_SERVICE_DISABLE_REGISTRATION=false GOGS_SERVICE_SHOW_REGISTRATION_BUTTON=true
```

Comment:
Actually the documentation recommends using your way. Could you please add '\' after each env?